modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayBuildAdministrator.testcase

```diff
@@ -240,11 +262,18 @@ definition {
 	@priority = "5"
 	test AdminCanNotAddDuplicateBuild {
 		task ("Given that the user add a new build") {
+			ApplicationsMenu.gotoSite(site = "Testray 2");
+
+			Testray.changePagination(
+				numberItems = "20",
+				valuePagination = "150");
+
 			Testray.goToProjectPage(projectName = "Projects_1 Project");
```

Comment:
Hi @joelrocha68, I think these steps can be added in setup, as it is repeated in all tests